### PR TITLE
Update node-mongo

### PR DESCRIPTION
**Pull request by Oigen43**

### CODE REVIEW

File: src/outbox/outboxService.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  ): Promise<OutboxEvent[] | null> {
+    await this.waitForConnection();
+    const collection = await this.getCollection(collectionName);
+    if (!collection) {
+      return null;
+    }
+
+    const events: OutboxEvent[] = data.map((e) => ({
+      _id: generateId(),
+      createdOn: DateTime.utc().toJSDate(),
```
Comment (KuhArt) [inline src/outbox/outboxService.ts:110]: Do we need luxon for that?

File: bin/publish.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+npm run n && npm run build && npm publish --registry https://npm.getpaperclip.com
```
Comment (KuhArt) [inline bin/publish.sh:2]: Is it a normal name for the registry? 
> npm.getpaperclip.com

File: package.json [hunk 3]
```diff
@@ -1,33 +1,50 @@
...
   "private": false,
-  "main": "src/index.js",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "test": "run-s test:*",
-    "test:eslint": "eslint ./",
-    "test:mocha": "NODE_ENV=test mocha --exit --recursive -c -R spec src/index.test.js"
+    "tests": "./bin/run-tests.sh",
+    "n": "run-s test:*",
```
Comment (KuhArt) [inline package.json:11]: Can you make 'n' command more declarative?

File: src/config/index.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable global-require */
+import * as _ from 'lodash';
+
+export interface Config {
```
Comment (KuhArt) [inline src/config/index.ts:4]: I just think that the config can be moved to a separate npm package in the future. 
Just an idea

File: src/database.ts [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    await this.waitForConnection();
+
+    if (!this.db) {
+      throw new Error('The db object is not defined');
+    }
+
+    try {
+      await this.db.createCollection<T>(name, opt.collectionCreateOptions || {});
+    } catch (error) {
+      if (error instanceof MongoError && error.code === 48) {
```
Comment (KuhArt) [inline src/database.ts:122]: What does this code mean?

**Review by KuhArt — CHANGES_REQUESTED** [review]

File: src/database.ts (hunk 5 above)
Comment (Oigen43) [reply src/database.ts:122]: It means that collection is already exists

**Review by KuhArt — APPROVED** [review]

**Pull request merged**